apollo/handlers/user.py

```diff
@@ -62,9 +75,7 @@ def patch_user(user_data: UpdateUserSchema, request: Request,
         data.pop('password'))
     user.has_changed_initial_password = True
```

Comment:
I'd lift this out of this method to not modify the argument. Just do it at the callsite.